packages/appstore/src/components/add-more-wallets/add-more-wallets.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Text, Loading } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { useAvailableWallets } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import CarouselContainer from './carousel-container';
+import AddWalletCard from './wallet-add-card';
+
+import './add-more-wallets.scss';
+
+const AddMoreWallets = observer(() => {
+    const { data, isLoading } = useAvailableWallets();
+    const { ui: is_mobile } = useStore();
+
+    if (!data) return <Loading is_fullscreen={false} />;
+
+    return (
+        <div className='add-wallets'>
+            <Text as='h2' size={is_mobile ? 'm' : 'l'} color='prominent' weight='bolder' className='add-wallets__title'>
+                {localize('Add more Wallets')}
+            </Text>
+            <CarouselContainer>
+                {isLoading ? (
+                    <Loading is_fullscreen={false} />
+                ) : (
```

Comment:
this is the one yashim is referring to, do we need to have the first loading check above since we already have it here?

Reply:
@maryia-deriv I don't think we need two Loading here, We can show static html even if the API is still in progress.
Would there be case where we have data and isLoading is also true ?